src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -72,6 +72,7 @@
             = DTLS_PREDICATE.negate();
 
     public static final PacketDelayStats packetDelayStats = new PacketDelayStats();
+    private final BridgeJitterStats bridgeJitterStats = new BridgeJitterStats();
```

Comment:
Can we go back to following the convention for java? Add docs on the fields (I mention it now because I think a doc on this one will be actually helpful), and keep the statics on top.

Reply:
yeah will fix this